scripts/us_census/enhanced_tmcf/data/output/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
alternatively, add /output and /zip_download to the .gitignore in the /data folder. (and only create the dirs if not present already prior to download.

Reply:
done